app/controllers/application_controller.rb

```diff
@@ -48,7 +48,15 @@ def peek_enabled?
   private
 
   def current_user
-    @current_user ||= User.find_or_create_by(email_address: session[:user]['email']) if session[:user] && session[:user]['email']
+    @current_user ||= find_current_user
+  end
+
+  def find_current_user
+    if ENV['FORCE_USER_ID']
```

Comment:
I've used it on 3 different branches in the last 24 hours. It's very helpful ime.

Point well taken about data privacy however.

I think we really need some secure way to act-as-another-user, in order to debug authz rules, display issues, etc. I'm very open to suggestions. It's just a basic debugging/testing aide, imo, to be able to reproduce.

I can probably use/extend `lib/populator.rb` to do more of the dev work; for debugging, though, I'm not sure how best to solve this need.